### ISSUE TITLE
Menu Flyout Acrylic doens't work in Windows 11

### ISSUE THREAD

**Issue by RaphMar2021** [microsoft/microsoft-ui-xaml#8396]
### Describe the bug

The flyout should show an acrylic effect, like in Edge.
![image](https://user-images.githubusercontent.com/81994075/232336872-2d4d8e85-ece2-449a-8e4b-e2d506d26a9f.png)

But the acrylic effect doens't show in my app's flyouts
![image](https://user-images.githubusercontent.com/81994075/232336931-37051b40-33f8-4694-bad2-e673f817cd26.png)

### Steps to reproduce the bug

1 : Create a WinUI 3 app
2 : Add a basic menu bar with flyouts

### Expected behavior

It should show an Acrylic effect

### Screenshots

Edge : 
![image](https://user-images.githubusercontent.com/81994075/232336979-2aec516c-a353-479a-acbe-d383a334bbfc.png)

WinUI 3 : 
![image](https://user-images.githubusercontent.com/81994075/232336961-b2e0694f-fa0f-4305-a203-5fa4bd56b287.png)

### NuGet package version

WinUI 3 - Windows App SDK 1.2.5: 1.2.230313.1

### Windows version

Windows 11 (22H2): Build 22621

### Additional context

It's only on Windows 11 since i tried the same application on a Windows 10 computer and the acrylic effect was working.

**Comment by Jay-o-Way** [comment]
Hi. Got some example code to reproduce? :)

**Comment by RaphMar2021** [comment]
Here's one (put it in your window's grid) : 

```xaml
<MenuBar>
    <MenuBarItem Title="File">
        <MenuFlyoutItem Text="New"/>
        <MenuFlyoutItem Text="Open..."/>
        <MenuFlyoutItem Text="Save"/>
        <MenuFlyoutItem Text="Exit"/>
    </MenuBarItem>
<MenuBar>
```

**Comment by zhuxb711** [comment]
I think this issue will only present if you enabled Acrylic/Mica as backdrop. Try replace the backdrop with a picture and background works again on menu flyout.

**Comment by Jay-o-Way** [comment]
@RaphMar2021 I understand how to construct a menu. I was hoping you could show more of the app. Like the namespaces or settings. Like said: where is the definition of (the background of) the menu/flyout?

**Comment by l619534951** [comment]
This scene is appear on acrylic backdrop. my temporary solution:   
```xaml
<MenuFlyout.MenuFlyoutPresenterStyle>
    <Style BasedOn="{StaticResource DefaultMenuFlyoutPresenterStyle}" TargetType="MenuFlyoutPresenter">
        <Setter Property="Background" Value="{ThemeResource CardBackgroundFillColorDefaultBrush}" />
    </Style>
</MenuFlyout.MenuFlyoutPresenterStyle>
```

**Comment by RaphMar2021** [comment]
Yes, but how i get the MenuFlyout of a MenuBarItem?

**Comment by l619534951** [comment]
ResetStyle.xaml
```xaml
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
    <ResourceDictionary.MergedDictionaries>
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
    </ResourceDictionary.MergedDictionaries>

    <Style BasedOn="{StaticResource DefaultMenuFlyoutPresenterStyle}" TargetType="MenuFlyoutPresenter">
        <Setter Property="Background" Value="{ThemeResource CardBackgroundFillColorDefaultBrush}" />
    </Style>
</ResourceDictionary>
```

in App.xaml
```xaml
···
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
                <ResourceDictionary Source="ms-appx:///Styles/ResetStyle.xaml" />
···
```

**Comment by Nevodev** [comment]
> This scene is appear on acrylic backdrop, and it looks different from WinUI2, which looks even better. my temporary solution:
> 
> ```
> <MenuFlyout.MenuFlyoutPresenterStyle>
>     <Style BasedOn="{StaticResource DefaultMenuFlyoutPresenterStyle}" TargetType="MenuFlyoutPresenter">
>         <Setter Property="Background" Value="{ThemeResource CardBackgroundFillColorDefaultBrush}" />
>     </Style>
> </MenuFlyout.MenuFlyoutPresenterStyle>
> ```
> 
> it looks like this now (light theme and dark theme): ![chnge](https://user-images.githubusercontent.com/17621099/232785985-7f2ba74e-6833-45cd-96b0-41e609a23847.png)

这个是什么clash客户端 wow

**Comment by l619534951** [comment]
> 这个是什么clash客户端 wow

自己用的，没有发布

**Comment by gegao18** [comment]
Hi @RaphMar2021, this should be fixed in WinUI 3 1.4. Here's what I see in the WinUI 3 Gallery:

<img width="638" alt="image" src="https://github.com/microsoft/microsoft-ui-xaml/assets/45014232/aa3c729c-068a-453c-9799-e337b2b5c64d">

<img width="522" alt="image" src="https://github.com/microsoft/microsoft-ui-xaml/assets/45014232/7926d610-9427-4629-b6bf-b4ded2579769">
